React capstone project - Metrics webapp

Mutai-Gilbert opened the pull request:
In this pull request I was able to:
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

Mutai-Gilbert committed:
Added changes to the LICENSE.md file

Mutai-Gilbert committed:
Added new link to the README.md file

Alfredbis29 reviewed (changes requested):
#  STATUS: REQUIRED CHANGES 
 ## Hi @alvaroiporre ,
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/dbe642c07a12e38cdc021eb826931557cc1eb657023c3609cc10e8f3b47f06d7/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c344a7a686a56435169576765414f46612f67697068792e676966) 
 ## Highlights 
 :heavy_check_mark: Linters are passing.
:heavy_check_mark: You have a good code structure.
 
 ## Required Changes 

- [x] I'm not able to inline in your Readme file but I noticed that the M.I.T license has been broken in your code, and this is something that needs to be addressed before we can proceed. As you may know, the M.I.T license is a widely used open-source license that allows others to use, modify, and distribute your work with certain conditions. By breaking this license, you may be inadvertently infringing on the rights of others or limiting their ability to use your work in accordance with the license terms. kindly fix this :point_down: 
![Screenshot from 2023-04-21 12-24-02](https://user-images.githubusercontent.com/88894525/233623084-73bb2f11-bc78-497d-8bca-eb7d33614730.png)

- [x] You did a good video presentation so far but you forget to highlight about a piece of the code that you find interesting or you feel proud of making. kindly flag up the codes you made during this project and you feel proud of :wink:  , you did a great job try to present also the codes you made so far in this [demo](https://www.loom.com/share/dc12eea05d2744809a69d3cd4aca9619)

- [x]  Please deploy your project as it helps showcase your work, gain valuable feedback, and identify any issues. kindly deploy your project to match the requirements for this project :wink: 

 
 ## Optional suggestions 

- [ ] N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mutai-Gilbert commented:
@Alfredbis29  Thank you for the Review

- [x] The MIT link was working and it's still working.
- [x] I have made changes to the Video presentation.
- [x] The project had been deployed before and it's still deployed Kindly recheck on your end.

Mutai-Gilbert committed:
Made changes to the README.md file: Added a new Video Link and made changes to the MIT Link

Mutai-Gilbert committed:
Updated the Loom Video Presentation 

Added a new descriptive loom video as suggested by the code reviewer.

Mutai-Gilbert committed:
Update the Video Description 

In the commit I have a new link as an update to the video description as suggested by the code reviewer.

BenMukebo commented on README.md:
```diff
@@ -226,5 +225,3 @@ Feel free to give it a start
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
..

BenMukebo reviewed (approved):
...

BenMukebo commented:
Hello @Mutai-Gilbert where did you put the video link, please add it

Mutai-Gilbert committed:
Update the Video and Demo Link

Added a new demo and video link.

Mutai-Gilbert committed:
Update Live Demo

Added a new Link to the Live demo.

BenMukebo commented on README.md:
```diff
@@ -61,8 +61,11 @@ This is a react diseases project that renders the covid updates from different c
-- [Live Demo Link]()
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- [OPTIONAL] Great job making the changes requested by the previous reviewer 👏🏻 But, it would also be better to correct the `issues page` link from the "Contributing" section to the correct page 🧑‍🔧

Mutai-Gilbert committed:
Merge branch 'dev' into Capstone

Mutai-Gilbert commented on README.md:
```diff
@@ -62,8 +62,7 @@ This is a react diseases project that renders the covid updates from different c
-
+- [Live Demo Link](https://www.loom.com/share/bf8adf1ace3d4b3caf857bad0e366b46)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Done

Mutai-Gilbert commented on README.md:
```diff
@@ -62,8 +62,7 @@ This is a react diseases project that renders the covid updates from different c
-
+- [Live Demo Link](https://www.loom.com/share/bf8adf1ace3d4b3caf857bad0e366b46)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Done

BenMukebo replied:
- [OPTIONAL] Hi @Mutai-Gilbert, Great job so far 👍🏻 , however, according to the design requirement:

![image](https://user-images.githubusercontent.com/77069011/221369107-72f482f6-c56b-4a37-a40a-f351658c7383.png)

According to the design, these are the possible colors and fonts:

![image](https://user-images.githubusercontent.com/77069011/221369223-bef67be4-996f-4179-bf9e-90bd1da74887.png)

You have the font right, however, the colors don't match. Please consider changing it to meet this requirement! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/221369335-7a0335d3-7303-4929-8bcd-09f2bfbdc75d.png) | ![Screenshot from 2023-04-21 15-55-08](https://user-images.githubusercontent.com/79058364/233662247-48ad451c-9d9a-4952-b66e-e3306ebdf7c6.png)|
| ![image](https://user-images.githubusercontent.com/77069011/221369479-434dc0b4-36a9-48d7-b867-a9e1877b1968.png) |![Screenshot from 2023-04-21 15-54-28](https://user-images.githubusercontent.com/79058364/233662255-acc9f7ef-7b1c-4d77-b993-d90119a8f37d.png)



Also, If you look closely, in your home page, the elements, in your case teams have a "chess-like" color design, if the left side is light, the right side should be darker and the inverse. Please consider applying this as suggested by the project requirements! 🤓 🔧

BenMukebo reviewed (approved):
# Status: Approved  :champagne:  ✔️:confetti_ball: 

## Hi @Mutai-Gilbert, 

### **Congratulations! 🎉 Project Approved ✅**

Good job so far. Well Done.👍🏻 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight 🎉
- Git flow is followed. :heavy_check_mark:
- You have followed  JavaScript best practices. ✔️ 
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good presentation video. ✔️ 
- Your app has been well deployed heavy_check_mark

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMukebo commented on README.md:
```diff
@@ -61,7 +61,8 @@ This is a react diseases project that renders the covid updates from different c
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-- [Live Demo Link](https://www.loom.com/share/bf8adf1ace3d4b3caf857bad0e366b46)
+- [Live Demo Link](https://react-diseases.onrender.com)
+
```
.
